gem/tests/test_profiles.py

```diff
@@ -0,0 +1,24 @@
+from molo.core.tests.base import MoloTestCaseMixin
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+
+
+class GemRegistrationViewTest(TestCase, MoloTestCaseMixin):
+    def setUp(self):
+        self.client = Client()
+        self.mk_main()
+
+    def test_user_info_displaying_after_registration(self):
+        self.user = User.objects.create_user(
+            username='tester',
+            email='tester@example.com',
+            password='tester')
+        self.user.profile.gender = 'female'
```

Comment:
This is the problem - you should be setting the value to `f` instead of `female`